docs/src/modules/components/backers.md

```diff
@@ -4,9 +4,9 @@ The core of Material-UI is a [crowd-funded](/discover-more/backers/) open-source
 
 ### Diamond 💎
 
-*3/3 slots available*
+_3/3 slots available_
```

Comment:
```suggestion
*3/3 slots available*
```

Reply:
Grrr. I wish VS Code plugins would do what they claim, and use the project's config files.